cypress/integration/support-mail.spec.ts

```diff
@@ -0,0 +1,15 @@
+describe("Support Mail", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/dashboard");
+  });
+
+  context("desktop resolution", () => {
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
+    });
+
+    it("opens the mail app", () => {
+      cy.get("nav").contains("Support").click({ force: true });
```

Comment:
I don't think you can test that the email app actually opens. What you can do though is check that the link has the correct `href`